api/handlers.go

```diff
@@ -0,0 +1,25 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/ONSdigital/dp-sessions-api/session"
+	"github.com/ONSdigital/log.go/log"
+	"net/http"
+)
+
```

Comment:
Going put this now to save you an arse-ache at the end - add Godoc for your public methods/functions are you go along.